pkg/coderefs/coderefs.go

```diff
@@ -328,11 +345,6 @@ func (g searchResultLines) makeReferenceHunksReps(projKey string, ctxLines int)
 			continue
 		}
 
-		if len(hunks) > maxHunksPerFileCount {
-			log.Warning.Printf("found %d code references in %s, which exceeded the limit of %d, truncating file hunks", len(hunks), fileSearchResults.path, maxHunksPerFileCount)
-			hunks = hunks[0:maxHunksPerFileCount]
-		}
```

Comment:
I removed this limit because it seemed a bit restrictive - combination of other existing limits should still be tight enough